tests/scripts/basic-build-test.sh

```diff
@@ -39,6 +39,10 @@ fi
 CONFIG_H='include/mbedtls/config.h'
 CONFIG_BAK="$CONFIG_H.bak"
 
+# Step 0 - print build environment info
+scripts/print_dev_env.sh
```

Comment:
Wrong filename - and in answer to @andresag01's comment - a check would require us to define correct versions, which is harder than it sounds. Nice idea though.